src/popover/internal.tsx

```diff
@@ -141,7 +143,7 @@ function InternalPopover(
       className={clsx(popoverClasses, styles['popover-content'])}
       data-awsui-referrer-id={referrerId}
     >
-      {visible && (
+      {visible && !__hidden && (
```

Comment:
On the API discussion, there was a question if this private API in the scope of this feature or not. Please discuss with the team before pushing it.

Reply:
Removed this for now. We can add it later if really needed.